docs/search_api/api.rst

```diff
@@ -0,0 +1,6 @@
+API Documentation
+*****************
+
+.. automodule:: rcsbapi.search
+    :members:
+    :special-members: __init__
```

Comment:
```suggestion
    :special-members: __init__
    
    .. autoclass:: rcsbapi.search.search_schema.SearchSchemaGroup
    :members: search, get_attribute_details, get_attribute_type

    .. autoclass:: rcsbapi.search.search_query.Session
        :members: rcsb_query_editor_url, rcsb_query_builder_url
```

I noticed that some of the methods we mention in the documentation aren't included in the generated page (rcsb_query_editor_url, get_attribute_details). I think it's because it's referencing __all__ which doesn't include SearchSchemaGroup and Session.

I don't think we should add those classes to __all__, so I think we can include the mentioned methods like this. But I wasn't able to test it with readthedocs, so the code may need some tweaking